optuna/visualization/_pareto_front.py

```diff
@@ -22,6 +26,15 @@
 _logger = optuna.logging.get_logger(__name__)
 
 
+class _ParetoFrontInfo(NamedTuple):
+    n_targets: int
+    target_names: List[str]
```

Comment:
[question]

`List` and `Sequence` both appear in this file. Do you have any policy to distinguish uses of them?